src/test/mochitest/browser_dbg-xhr-breakpoints.js

```diff
@@ -0,0 +1,19 @@
+add_task(async function() {
+    const dbg = await initDebugger("doc-xhr.html");
+    await waitForSources(dbg, "fetch.js");
+    await dbg.actions.setXHRBreakpoint("doc", "GET");
+    debugger;
```

Comment:
Is this supposed to be here?  Shouldn't the pause happen from setting the breakpoint?

Reply:
Yep, it should not be here